pkg/csplugin/broker.go

```diff
@@ -406,8 +406,14 @@ func getProcessAtr(username string, groupname string) (*syscall.SysProcAttr, err
 	}
 	g, err := user.LookupGroup(groupname)
 	if err != nil {
-		return nil, err
+		if groupname == "nogroup" {
```

Comment:
this part is too hackish to be fixed in code. let's deal with this specific part in packaging please :)